public/pages/Dashboard/containers/Dashboard.test.js

```diff
@@ -104,4 +193,26 @@ describe('Dashboard', () => {
     expect(wrapper.instance().state.alerts[0].alert_history).toStrictEqual([]);
     expect(wrapper.instance().state.alerts[0].error_message).toBe('');
   });
+
+  test('able to select single alert in flyout', () => {
```

Comment:
Looks like this test is just testing the render situation for a flyout when `perAlertView` is not set.

Reply:
Yes, I found out this is incomplete and will need some time to figure out the right way to perform the user events. Skipping this test with added TODO for now.